Cargo.toml

```diff
@@ -74,20 +67,31 @@ futures = "0.3.29"
 bytes = "1.5.0"
 dashmap = "5.5.3"
 redis_pool = { version = "0.2.1", optional = true }
-redis = { version = "0.23.3", features = ["aio", "tokio-comp"], optional = true }
+redis = { version = "0.23.3", features = [
+    "aio",
+    "tokio-comp",
+], optional = true }
 surrealdb = { version = "1.0.0", optional = true }
-aes-gcm = { version = "0.10.3"}
-base64 = { version = "0.21.5"}
-rand = { version = "0.8.5"}
-fastbloom-rs = {version = "0.5.7", optional = true }
+aes-gcm = { version = "0.10.3" }
+base64 = { version = "0.21.5" }
+rand = { version = "0.8.5" }
+fastbloom-rs = { version = "0.5.7", optional = true }
 mongodb = { version = "2.7.1", optional = true }
 
 [dev-dependencies]
-axum = { version = "0.6.20", features = ["macros"]}
+axum = { version = "0.6.20", features = ["macros"] }
 hyper = "0.14.27"
 tower = "0.4.13"
 log = { version = "0.4.20", default-features = false }
 
 [package.metadata.docs.rs]
-features = ["sqlite-rustls", "mysql-rustls", "redis-db", "surrealdb-rocksdb", "surrealdb-mem", "key-store", "advanced"]
+features = [
+    "sqlite-rustls",
+    "mysql-rustls",
+    "redis-db",
+    "surrealdb-rocksdb",
```

Comment:
you will need to remove these too then and set it to just surreal.

Reply:
I've applied the changes.